utilities_general.py

```
@@ -1,3 +1,1553 @@
+    try:
+        addcolumn = "ALTER TABLE Anexo_Eletronico_Obscat ADD COLUMN ordem INTEGER NOT NULL DEFAULT 0"
+        cursor.custom_execute(addcolumn, None, False, False)
+        commit = True  
+        obscats = "SELECT id_obscat FROM Anexo_Eletronico_Obscat"        
+        cursor.custom_execute(obscats)
+        obscats = cursor.fetchall()
+        ordem = 0
+        for obscat in obscats:
+            updateinto2 = "UPDATE Anexo_Eletronico_Obscat set ordem = ? WHERE id_obscat = ?"
+            cursor.custom_execute(updateinto2, (ordem, obscat[0],))
+            ordem += 1
```

Ruff

Use `enumerate()` for index variable `ordem` in `for` loop

(SIM113)